rllib/evaluation/sampler.py

```diff
@@ -51,14 +55,49 @@ class _PerfStats:
     def __init__(self):
         self.iters = 0
         self.env_wait_time = 0.0
-        self.processing_time = 0.0
+        self.pre_processing_time = 0.0
+        self.storing_samples_in_buffer = 0.0
         self.inference_time = 0.0
+        self.action_processing_time = 0.0
+        self.get_ma_train_batch = 0.0
+        self.postprocess_trajectories_so_far = 0.0
 
     def get(self):
+        # Mean multiplicator (1000 = ms -> sec).
+        factor = 1000 / self.iters
         return {
-            "mean_env_wait_ms": self.env_wait_time * 1000 / self.iters,
-            "mean_processing_ms": self.processing_time * 1000 / self.iters,
-            "mean_inference_ms": self.inference_time * 1000 / self.iters
+            # Waiting for environment (during poll).
```

Comment:
Oof, this is a lot of stuff to expose to users. Could we limit it to at most 4-5 stats? Maybe a lot of the detailed breakdowns can be hidden by feature flag, or omitted entirely.

Reply:
Yeah, true, I'll remove the totals again. These were just for learning about which parts are bottlenecks.

Reply:
done